app/views/parent_objects/_form.html.erb

```diff
@@ -180,6 +180,11 @@
       <%= form.label :preservica_uri %>
       <%= form.text_field :preservica_uri %>
     </div>
+
+    <div class="col-med-3">
+      <%= form.label :preservica_representation_type %>
+      <%= form.select(:preservica_representation_type, ['None', 'Preservation', 'Access'], {}, style: "width:178px;" ) %>
```

Comment:
Could you remove the 'None' option, please?  A preservica parent must have either Preservation or Access, no other values are valid.